### PR TITLE
Flask sqlalchemy psycopg2 integration

### PR DESCRIPTION
**Pull request by Thiyagu55**
# Description

Integrating flask sqlcommenter into psycopg2 and sqlalchemy

This can be achieved by enabling enable_commenter flag inside Flask instrument initialization and also in SQLAlchemy Instrumentation  or psycopg2 instrumentation

```python
FlaskInstrumentor().instrument_app(app, tracer_provider=provider, enable_commenter=True)

# For sqlalchemy
SQLAlchemyInstrumentor().instrument( enable_commenter=True, engine=engine)

# For psycopg2
Psycopg2Instrumentor().instrument(tracer_provider=tracer_provider, enable_commenter=True)
```

- The flask instrumentation will append the metrics in opentelemetry context in key name SQLCOMMENTER_VALUES
- The SQLAlchemy ORM or psycopg2 driver will read the context and then append these info to the sql query logs whenever the sql query is made

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)
- [X] This change requires a documentation update

# How Has This Been Tested?

- [X] instrumentation/opentelemetry-instrumentation-flask/tests/test_sqlcommenter.py
- [X] instrumentation/opentelemetry-instrumentation-dbapi/tests/test_dbapi_integration.py test_executemany_flask_integration_comment
- [X] instrumentation/opentelemetry-instrumentation-sqlalchemy/tests/test_sqlcommenter.py test_sqlcommenter_flask_integration

# Does This PR Require a Core Repo Change?

- [X] No.

# Checklist:

See [contributing.md](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/CONTRIBUTING.md) for styleguide, changelog guidelines, and more.

- [X] Followed the style guidelines of this project
- [X] Changelogs have been updated
- [X] Unit tests have been added
- [X] Documentation has been updated

### CODE REVIEW

**Comment by Thiyagu55** [comment]
@srikanthccv For your visibility

File: instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py [hunk 1]
```diff
@@ -430,6 +431,18 @@ def traced_execution(
                     ):
                         commenter_data.update(**_get_opentelemetry_values())
 
+                    # Add flask related tags
+                    sqlcommenter_flask_values = (
+                        opentelemetry_context.get_value(
+                            "SQLCOMMENTER_FLASK_VALUES"
```
Comment (srikanthccv) [inline instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py:437]: Is it possible to move this framework specific things to commenter code and remove these references from here.

**Comment by srikanthccv** [comment]
I like the updated approach better. I will take a closer look again.

File: CHANGELOG.md [hunk 2]
```diff
@@ -26,6 +26,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#1206](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1206))
 - Add psycopg2 native tags to sqlcommenter
   ([#1203](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1203))
+- Flask sqlalchemy psycopg2 integration
+  ([#1224](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1224))
```
Comment (srikanthccv) [inline CHANGELOG.md:30]: Please move this to unreleased

**Review by srikanthccv — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
```
Comment (srikanthccv) [inline CHANGELOG.md:9]:
```suggestion
```

**Review by srikanthccv — APPROVED** [review]

**Pull request merged**